dev_docs/tutorials/adding_performance_metrics.mdx

```diff
@@ -7,76 +7,156 @@ date: 2022-07-07
+### Simple performance events
+
+Lets assume we intend to report the performance of a specific action called `APP_ACTION`.
+In order to do so, we need to first measure the timing of that action.
+Once we have the time measurment, we can use the `reportPerformanceMetricEvent` API to report it.
+
+The most basic form of reporting would be:
 
+```typescript
 reportPerformanceMetricEvent(analytics, {
-  eventName: KIBANA_LOADED_EVENT,
-  meta: {
-    // fields that are searchable-as-keywords but not aggregateable
-    kibana_version: this.coreContext.env.packageInfo.version,
-    protocol      : window.location.protocol,
-  },
-  duration: timing[LOAD_FIRST_NAV],
-  key1    : LOAD_START,
-  value1  : timing[LOAD_START],
-  key2    : LOAD_BOOTSTRAP_START,
-  value2  : timing[LOAD_BOOTSTRAP_START],
-  key3    : LOAD_CORE_CREATED,
-  value3  : timing[LOAD_CORE_CREATED],
-  key4    : LOAD_SETUP_DONE,
-  value4  : timing[LOAD_SETUP_DONE],
-  key5    : LOAD_START_DONE,
-  value5  : timing[LOAD_START_DONE],
+  eventName: APP_ACTION,     
+  duration,                  // Duration in milliseconds
 });
+```
+
+Once executed, the metric would be delivered to the `stack-telemetry` cluster, along side with the event's context.
+The data is updated periodically, so you might have to wait up to 30 minutes to see your data in the index.
+
+Once indexed, this metric will appear in `ebt-kibana` index. 
+It we also be mapped into an additional index, dedicated to performance metrics.
+We recommend using the performance metrics indices, where you can expect the following structure:
+
+```typescript
+{
+  "_index": "backing-ebt-kibana-browser-performance-metrics-000001",    // Performance metrics are stored in a dedicated simplified index (browser \ server). 
+  "_source": {
+    "timestamp": "2022-08-31T11:29:58.275Z"
+    "event_type": "performance_metric",                                 // All events share a common event type to simplify mapping
+    "eventName": "dashboard_loaded",                                    // Event name as specified when reporting it
+    "duration": 736,                                                    // Event duration as specified when reporting it
+    "context": {                                                        // Context holds information identifying the deployment, version, application and page that generated the event
+      "version": "8.5.0-SNAPSHOT",
+      "applicationId": "dashboards",
+      "page": "app",
+      "entityId": "61c58ad0-3dd3-11e8-b2b9-5d5dc1715159",
+      "branch": "main",
+      "labels": {
+        "journeyName": "flight_dashboard",
+        ...
+      }
+      ...
+    },
+    ...
+  },
+}
+```
+
+### Performance events with breakdowns and metadata
+
+Lets assume we are interested in benchmarking the performance of a more complex event `COMPLEX_APP_ACTION`, that is made up of two steps: 
+ - `STEP_A` where there are two possible flows `flow1` or `flow2` 
```

Comment:
These steps and flows are a bit ambiguous imho, maybe we can provide example what can be those since in the example [here](https://github.com/elastic/kibana/pull/139852/files#diff-779dac43cbf7efbd7867f94bf1b4e300db3e2fabf654825752b6d377df76d5d9R93)it's also constant and it might be hard to understand what can be step and flow.

Reply:
I made the example a bit more specific